java/src/jmri/jmrit/consisttool/ConsistToolFrame.java

```diff
@@ -616,28 +611,18 @@ public void dispose() {
         consistManager.removeConsistListListener(this);
     }
 
-    // want to read the consist file after the consists have been loaded
-    // from the command station.  The _readConsistFile flag tells the
-    // notifyConsistListChanged routine to do this.  notifyConsistListChanged
-    // sets the value to false after the file is read.
-    private boolean _readConsistFile = true;
```

Comment:
Is this not needed anymore? I don't use consists so I don't know how this works.